contracts/controller/src/test_utils.rs

```diff
@@ -1,14 +0,0 @@
-// use near_sdk::AccountId;
```

Comment:
@dmytro-shash I suppose this file should resolve problem with DRY for `init_test_env`. This was commented out because it's breaks compilation but should be fixed and correctly used inctead of duplicating `init_test_env` everywhere